src/Search/Api/Elasticsearch/Command/resources/labs-post.yaml

```diff
@@ -1,35 +1,5 @@
 labs-post:
   properties:
-    content:
```

Comment:
does this mean the same information goes into another field defined here or simply that we don't define these fields explicitly?